indico/modules/events/sessions/controllers/display.py

```diff
@@ -72,7 +73,10 @@ def _process(self):
 @allow_signed_url
 class RHExportSessionToICAL(RHDisplaySessionBase):
     def _process(self):
-        return send_file('session.ics', get_session_ical_file(self.session), 'text/calendar')
+        detailed = request.args.get('detail', 'sessions') == 'contributions'
```

Comment:
```suggestion
        detailed = request.args.get('detail') == 'contributions'
```

The default value is no longer needed